### PR TITLE
Trivialize snapping redirects (^C, experimental)

### PR DESCRIPTION
**Pull request by cschanaj**
No ruleset needed to be excluded.

### CODE REVIEW

File: src/chrome/content/rules/CuidadoDeSalud.gov.xml [hunk 1]
```diff
@@ -5,6 +5,5 @@
 	<securecookie host="www\.cuidadodesalud\.gov"
 			name=".+" />
 
-	<rule from="^(?:http://(?:www\.)?|https://)cuidadodesalud\.gov/"
```
Comment (J0WI) [inline src/chrome/content/rules/CuidadoDeSalud.gov.xml]: Oh, that's weird. There are around 18 rules like this.

**Comment by J0WI** [comment]
For some reason there are `Disabled by https-everywhere-checker because:` comments in some rulesets, but this is unrelated to your change. Maybe another cleanup task.

**Comment by J0WI** [comment]
Sorry, I just discovered that there are some outdated comments in the ruleset, like `Cert only matches www` or `mismatch on ^`. I'm not sure if we can change them by script.

**Comment by cschanaj** [comment]
@J0WI I don't think removing the outdated comment can be done trivially by script. Unlike rewriting direct rewrite as in #10872, removing snapping rewrite actually introduce new rules to the code base . In some rare case (I don't have an example), it can be problematic.

**Pull request merged**